docs/tactics.md

```diff
@@ -1337,3 +1337,21 @@ See also additional documentation of `using_well_founded` in
 * `@[simps]` reduces let-expressions where necessary.
 * If one of the fields is a partially applied constructor, we will eta-expand it
   (this likely never happens).
+
+### mk_simp_set
+
+The command `mk_simp_set simp_name` creates a simp set with name `simp_name`.
+Lemmas tagged with `@[simp_name]` will be included when `simp using simp_name` is called.
+An optional description for the simp set can be provided with `mk_simp_set simp_name "description"`.
+
+Appending the command with `with id1 id2 ...` will use `id1`, `id2`, ... as dependencies
```

Comment:
This sentence is not clear to me. Do you mean that `id1` is an existing simp set, and that all lemmas in the `id1` simpset will also be included in the simp set you are creating? In this case, what about lemmas that you tag later on with `id1`: will they also be included in `simp_name`? Or is it something completely different?

Same thing in the docstring for `mk_simp_set`.

Reply:
To be honest, I don't know. It's passed as an argument named `attr_deps` to the tactic `mk_simp_attr`. This is in core and its behavior is undocumented, but it's used in mathlib.

I *think* it has to do with when the cache is regenerated. When you create a user attribute with a cache, you can give it a list of dependencies, which are other attributes whose caches must be computed first.

Reply:
Is there any way to test this? Or should we keep this line?

Reply:
Some experimenting implies that it does extend the simp set.
```lean
run_cmd mk_simp_attr `my_simp_attr
constants a b : ℕ
@[my_simp_attr] lemma l1 : a = 2 := sorry

run_cmd mk_simp_attr `my_new_simp_attr [`my_simp_attr]

example : a = 2 := by simp only with my_new_simp_attr
```
I'll update the docs to phrase this better.